dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/EventExporterTest.java

```diff
@@ -345,19 +329,15 @@ void testExportEventsWhenFilteringByDataElementsEqual( Function<EventSearchParam
 
     @ParameterizedTest
     @MethodSource( "getEventsFunctions" )
-    void testExportEventsWhenFilteringByDataElementsIn( Function<EventSearchParams, List<String>> eventFunction )
+    @Disabled( "This test is disabled because it doesn't look like we offer the feature to filter by multiple values with the like operator" )
```

Comment:
```suggestion
   
```

Reply:
`getEventsFunctions()` probably was in place to parametrize tests with multiple functions. I think you can get rid of it and use and helper method.